src/behavior.ts

```diff
@@ -127,23 +127,33 @@ export class Behavior {
         return selectedIds.some((value: ISelectionId) => value.includes(<ISelectionId>dataPoint.identity));
     }
```

Comment:
isDataPointSelected()...
why checking with value.includes, identity has _equals_ api for key comparison